resources/views/email/auth/register.blade.php

```diff
@@ -1,7 +1,11 @@
-Thanks for signing up!
+{{ trans('auth.register.email.body.intro') }}!
 
-We just need you to confirm your email address:
-{{ route('register.confirm', [$user->registration_token]) }}
+{!! trans('auth.register.email.body.text', [
+    'link' => link_to_route('register.confirm', null, [$user->registration_token], [])
```

Comment:
Lets not introduce Laravelcollective HTML facade dependency, <a href="{{ route(...with $absolute parameter...)"> is OK.